doc/source/serve/doc_code/direct_ingress.py

```diff
@@ -0,0 +1,28 @@
+# flake8: noqa
+
+# __begin_server__
+from ray import serve
+from ray.serve.drivers import DefaultgRPCDriver
+
+
+@serve.deployment
+class D1:
+    def __call__(self, input):
+        # Do something with input
+        return input["a"]
+
+
+my_deployment = DefaultgRPCDriver.bind(D1.bind())
+
+serve.run(my_deployment)
+# __end_server__
+
+from ray.serve.generated import serve_pb2, serve_pb2_grpc
```

Comment:
move this under `# __begin_client__` so user can copy the client code and it works`.